### ISSUE TITLE
Mind and fix the Dependabot security alerts

### ISSUE THREAD

**Issue by pmotch** [open-voice-network/website#44]
Dependabot has security alerts on the repo for dependancies that have known security issues. Should regularly look at them and apply fixes.

**Comment by pmotch** [comment]
@JohnTrammellTarget I noticed you took a stab at this; curious what you ran into. I was thinking about the best way to approach those dependabot PRs too.

**Comment by JohnTrammellTarget** [comment]
Some of the PRs merge cleanly, a lot of them do not. We might need to update the core victor-hugo application to get a consistent set of libraries (if possible).

**Comment by JohnTrammellTarget** [comment]
I tried a quick sync with the victor-hugo upstream, and it looks like *they* have version problems as well.

**Comment by pmotch** [comment]
Yeah, interesting. I noticed the same. Curious to know if they keep those up to date OR if we'll need to open some PRs there.
